### DISCUSSION TITLE
Zeek File Extraction - all mimetypes

### DISCUSSION THREAD

**Discussion by Talisec** [Security-Onion-Solutions/securityonion#3141] [Unsupported Versions] [answered]
I see there is a conversation at https://groups.google.com/g/security-onion/c/1vvTFq0c6Lc on the previous version of SO. What file needs to be modified in the current version to extract all mimetypes? Thanks in advance.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/3178
